js/components/ChooseAvatar/styles.js

```diff
@@ -51,20 +56,24 @@ const styles = StyleSheet.create({
     height: height / 9
   },
   editAvatar: {
+    marginLeft: 20,
+    marginRight: 20,
+    marginTop: 15,
+    marginBottom: 15,
     resizeMode: "contain",
     width: width / 4,
     height: height / 8
   },
   cameraIcon: {
     resizeMode: "contain",
-    width: 40,
-    height: 40,
+    width: width / 12,
+    height: height/ 25,
     position: "absolute",
-    bottom: -5,
-    right: -10,
+    bottom: 0,
+    right: 0,
```

Comment:
we try to avoid using top, bottom, left or right properties.

Reply:
I have to use bottom and right for position absolute.